lib/cloud_controller/opi/instances_client.rb

```diff
@@ -34,20 +38,11 @@ def ==(other)
     end
 
     def lrp_instances(process)
-      path = "/apps/#{process.guid}/#{process.version}/instances"
-      begin
-        retries ||= 0
-        resp = client.get(path)
-        resp_json = JSON.parse(resp.body)
-        handle_error(resp_json)
-      rescue CloudController::Errors::NoRunningInstances => e
-        sleep(1)
-        retry if (retries += 1) < 5
-        raise e
-      end
-      process_guid = resp_json['process_guid']
-      resp_json['instances'].map do |instance|
-        ActualLRP.new(instance, process_guid)
+      return confirm_stopped(process) if process.stopped?
```

Comment:
What calls this method? I'm wondering why it's trying to even get instances for a stopped process?

Reply:
cool sounds good 👍